### ISSUE TITLE
Create PieChart with Parameter Gradient results error messages (InverseColors)

### ISSUE THREAD

**Issue by ferwe** [EvotecIT/PSWriteHTML#42]
Hey there

I figured out, that there is a problem with pie charts.
```Powershell
 Panel {
                    Chart -Title "Tickets nach Eingangsdatum" {
                        New-ChartPie -Name 'Tickets 0 - 15 Days'    -Value $Object.Tickets.TicketCount0To15days.Count   -Color Green
                        New-ChartPie -Name 'Tickets 15 - 30 Days'   -Value $Object.Tickets.TicketCount15To30days.Count  -Color Orange
                        New-ChartPie -Name 'Tickets 30 - 50 Days'   -Value $Object.Tickets.TicketCount30To50days.Count  -Color OrangeRed
                        New-ChartPie -Name 'Tickets older 50 Days'  -Value $Object.Tickets.TicketCount50To100days.Count -Color Red
                    }
                }
```

If I use this code the following error Message results
```
true : The term 'true' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Program Files\WindowsPowerShell\Modules\PSWriteHTML\0.0.52\PSWriteHTML.psm1:1660 char:32
+             inverseColors    = true
+                                ~~~~
    + CategoryInfo          : ObjectNotFound: (true:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

The issue comes from a missing $ before "true" for the property "inverseColors" in psm1
There are multiple positions where this issue comes up.
I will fix this stuff and create a Pull Request.

Cheers

**Comment by PrzemyslawKlys** [comment]
ye, sorry for that. I thought I tested it and it worked. It needs fixing in two places. 

- New-ChartInternalGradient 
- New-ChartInternalRadialType1

**Comment by ferwe** [comment]
No worries.
I will have a look into it.
The module is awesome ;D

**Comment by ferwe** [comment]
Fixed in Pull Request #43
